examples/VREP/position_control_osc.py

```diff
@@ -28,39 +29,52 @@
 ee_track = []
 target_track = []
 
+# control (x, y, z) out of [x, y, z, alpha, beta, gamma]
+ctrlr_dof = [True, True, True, False, False, False]
+
 
 try:
     # get the end-effector's initial position
     feedback = interface.get_feedback()
     start = robot_config.Tx('EE', feedback['q'])
+
     # make the target offset from that start position
     target_xyz = start + np.array([0.2, -0.2, -0.3])
     interface.set_xyz(name='target', xyz=target_xyz)
 
-    # run ctrl.generate once to load all functions
-    zeros = np.zeros(robot_config.N_JOINTS)
-    ctrlr.generate(q=zeros, dq=zeros, target_pos=target_xyz)
-    robot_config.R('EE', q=zeros)
-
-    print('\nSimulation starting...\n')
-
     count = 0.0
+    print('\nSimulation starting...\n')
     while count < 1500:
         # get joint angle and velocity feedback
         feedback = interface.get_feedback()
+
+        target = np.hstack([
+            interface.get_xyz('target'),
+            interface.get_orientation('target')])
+
+        name = 'joint1'
+        vrep_angles = interface.get_orientation('UR5_%s' % name)
+        rc_matrix = robot_config.R(name, feedback['q'])
+        rc_angles = transformations.euler_from_matrix(rc_matrix, axes='sxyz')
+
+        interface.set_xyz('object', robot_config.Tx(name, q=feedback['q']))
+        interface.set_orientation('object', rc_angles)
```

Comment:
Why are we setting an orientation if we are only controlling x, y, and z. Alpha, beta, and gamma are all `False` in `ctrlr_dof`